### PR TITLE
hero setion in landing page

### LINKED ISSUE #31: [build] hero setion in landing page
create hero section which contains an image + brief statement about our website

### PR DESCRIPTION
**Pull request by mai-mohammed**
In this branch I did the following:

- create a hero section component with its styles.

### CODE REVIEW

**Review by abdou059 — APPROVED** [review]

File: client/src/components/heroSection/heroSection.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { ThemeProvider } from '@emotion/react';
+import useStyles, { theme } from './heroStyles';
+
+function HeroSection() {
+  const classes = useStyles();
+  return (
+
+    <section className={classes.heroSection}>
+      <Box
+        className={classes.heroImage}
```
Comment (LinaYahya) [inline client/src/components/heroSection/heroSection.tsx]: I'm really sorry to suggest using `makeStyles`, It seems deprecated in the current version SO may convert to `sx` instead!

File: client/src/components/heroSection/heroSection.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        component="img"
+        alt=""
```
Comment (LinaYahya) [inline client/src/components/heroSection/heroSection.tsx]: use real `alt` value

File: client/package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
+    "@material-ui/core": "^4.12.4",
```
Comment (LinaYahya) [inline client/package.json]: uninstall

File: .github/workflows/node.js.yml [hunk 4]
```diff
@@ -43,7 +43,7 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - name: Install dependencies
-        run: npm ci
+        run: npm config set legacy-peer-deps true &&  npm ci
```
Comment (LinaYahya) [inline .github/workflows/node.js.yml]: Why u add this?

**Review by LinaYahya — COMMENTED** [review]
Good Job Mai, 
Small changes and rename main component files to `index.tsx` and the folder to `UpperCase`

**Review by husamkamal — APPROVED** [review]

File: client/src/components/heroSection/heroSection.tsx (hunk 2 above)
Comment (mai-mohammed) [reply client/src/components/heroSection/heroSection.tsx]: I think this image is for decoration purposes so there is  no need to add alt  value

File: .github/workflows/node.js.yml (hunk 4 above)
Comment (mai-mohammed) [reply .github/workflows/node.js.yml:46]: see description

Comment (mai-mohammed) [reply .github/workflows/node.js.yml]: I added it to solve a problem that appeared when I installed @material-ui/core  package.

**Review by tariqabusada — APPROVED** [review]

**Pull request merged**